src/compas_model/elements/element.py

```diff
@@ -164,9 +164,21 @@ def material(self) -> Union[Material, None]:
         return self._material
 
     @property
-    def parent(self) -> "ElementNode":
+    def parent_node(self) -> "ElementNode":
         return self.treenode.parent
 
+    @property
+    def parent(self) -> "Element":
+        return self.parent_node.element
+
+    @property
+    def children_nodes(self) -> "list[ElementNode]":
+        return self.treenode.children
+
+    @property
+    def children(self) -> "list[Element]":
+        return [child.element for child in self.children_nodes]
```

Comment:
I tried to be a bit more explicit here:
`parent`, `children` give you `Element`s
`parent_node`, `children_nodes` give you `Node`s

Reply:
parentnode